content/working-groups/learning/minutes/2019.10.01-meeting.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Learning WG 2019-10-01 sync meeting
+type: docs
+---
+# Learning WG 2019-10-01 sync meeting
+
+- Team discussed that watching same video at the same time works well so far
+  - Everyone is [taking notes in this paper doc](https://paper.dropbox.com/doc/Ty-lecture-summary--AlwygwSe9rh1NVn2BB6CaBB9Ag-4jFj9bVOLlW7uhIOWHITX).
+  - This lecture https://github.com/rust-lang/rustc-guide/issues/375
```

Comment:
```suggestion
  - About this lecture https://github.com/rust-lang/rustc-guide/issues/375
```

Reply:
My suggestion also doesn't look great. When I read "this lecture" I thought what about this lecture? seems like an incomplete phrase.